packages/webpack-cli/lib/groups/BasicGroup.js

```diff
@@ -30,9 +30,8 @@ class BasicGroup extends GroupHelper {
             // const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
```

Comment:
We need to remove these lines, let's do it in other PR

Reply:
sure yes

Reply:
@evilebottnawi already handled here - https://github.com/webpack/webpack-cli/pull/1361

Will complete it later and refactor to its own package